### ISSUE TITLE
Animal ID Types with GroupLots

### ISSUE THREAD

**Issue by Arun45** [AAVLD-USAHA-ITStandards/eCVI#34]
There has been multiple requests from AHOs to add Animal Tags for the GroupLots as well. So we need to consider adding Animal Tags to GroupLots.

**Comment by mkm1879** [comment]
We would need to look at this use-case more closely.  Would the IDs be group-lot identifiers or are they trying to put individual IDs in a group?  The initial use-case for GroupLot was loads of feeder cattle that don't require individual ID.

**Comment by Deevet84** [comment]
There are actually 3 scenarios for group lots:
1. group lots that don't require individual ID's - like feeder steers
2. Group lots that have one ID like some  commercial poultry or swine
3. Group lots where a large group of like animals are moving but they all have individual ID's - breeding swine or dairy heifers/cows moving

**Comment by mkm1879** [comment]
#3 is one the committee never discussed.  They would be “unwound” into individual animals with similar signalment.  They can certainly be handled in a GUI application as a group the way you describe.

**Comment by SusanCulp** [comment]
Following up on our discussion today, Scenario 2 needs to account for the following:
One group movement made up of several grouplot ids with an unknown number of pigs associated with each grouplot id.
So we know the total number of animals and we know the grouplots contained within the movement.
We need to be able to account for more than one grouplot id associated with one group of animals.

**Comment by SusanCulp** [comment]
Sorry, forgot my signature.
Susan Culp, DVM
Veterinarian
Institute for Infectious Animal Diseases (IIAD)
A Member of the Texas A&M University System
 
1500 Research Parkway, Suite B270
College Station, TX 77843-2129
Office 979.458.2375 | Fax 979.845.6574

**Comment by mkm1879** [comment]
So, let me rephrase this discussion into a concrete proposal.  As I see it the one thing we can change to facilitate this is to change GroupLotID from maxOccurs="1" to maxOccurs="unbounded".
I think I would favor that change.  I don't believe it would truly "break" any receiving application.  Worst case scenario would be loss of additional IDs.
Others thoughts?

**Comment by rmunger** [comment]
USDA is unclear on the purpose of this discussion topic.  The eCVI schema needs to follow interstate movement regulations.  As a reminder we have included the regulatory definitions of Group/lot identification.

Per 9 CFR Part 86.1 Definitions:
Group/lot identification number (GIN).** The identification number used to uniquely identify a ‘‘unit of animals’’ of the same species that is managed together as one group throughout the preharvest production chain. When a GIN is used, it is recorded on documents accompanying the animals
moving interstate; it is not necessary to have the GIN attached to each animal.

In addition 9 CFR Part 71.19 Identification of Swine in Interstate Commerce contains the following requirements for group identification:
(c) Swine that are kept as a group are not required to be individually identified when in interstate commerce if:
(1) They were born on the same premises;
(2) They were raised on the same premises where they were born;
(3) They are moved in a group directly to a slaughtering establishment from the place where they were raised;
(4) They are not mixed with swine from any other premises, between the time they are born and the time they arrive at the slaughtering establishment; and
(5) They are slaughtered one after another, as a group, and not mixed with other swine at slaughter; or approved identification is applied to the swine after entry into the slaughtering establishment.
--

**Comment by rmunger** [comment]
Based on conversations following my last post we (VS) agree to move forward with Susan and Dee's request for group/lots support. 
I along with others were confused between group lots and groups of animals that fall under the use of a Group Identification Number (GIN) for official ID.

During our discussion it became clear that group/lots are not the same as groups of animals that fall GIN classification and we determined that the existing schema does not support the use and enforcement of a GIN as it does for other official ID types.
Therefore, we request that the schema be amended to support the GIN

**Comment by mkm1879** [comment]
The change on the board is simply to allow multiple group IDs in one group.  If you are proposing a set of named structures for GINs similar to the AIN, MfrRFID, etc. We will need those clearly spelled out--as regular expressions if possible.

**Comment by Deevet84** [comment]
Randy,

Thanks for your reply and help. We appreciate the call last week and the decision to allow our request from a USDA perspective. I am not sure who all is copied on the topic so I am individually including some interested parties on the cc line. It is my understanding that the necessary change is very simple for Mike Martin to make related to the actual schema change for allowing more than one Group lot ID to be included in the xml data push from a cvi. I am including Justin Smith for his awareness though as he is the chair of the committee that manages schema changes for the SAHO’s and stakeholders. I am not sure if there needs to be any more discussion from that group or we can move forward with this change? We are actually holding up the next push/update of our AgView  CVI app in the hopes this requested minor schema change can get finished quickly and we can include it. There are a lot of moving parts still out there including:


  *   Schema working group approves change
  *   Dr. Martin finalizes schema code change
  *   USA Herds and TraceFirst accommodate receipt of new data format
  *   AgView IT makes change internally
  *   Change is pushed to users

As to the schema change request for the GIN – that is a separate issue that I hope does not delay the needed implementation of the Group lot ID change. So Mike or Justin - should that change request from Randy be started on a separate github page. I am asking so that we can hopefully move our requested change forward quickly without getting bogged down in a GIN discussion. The github link is below. Thanks.

 https://github.com/AAVLD-USAHA-ITStandards/eCVI/issues/34

Dee

Dee Ellis, DVM MPA
Institute for Infectious Animal Diseases (IIAD)
Texas A&M University System
1500 Research Parkway, Suite B270
College Station, TX 77843-2129
Cell 512.560.9741
iiad.tamu.edu | @IIADnews

**Comment by mkm1879** [comment]
I believe this was closed as of v2.2
